src/client/components/SearchBar.jsx

```diff
@@ -19,9 +19,28 @@ class SearchBar extends React.Component {
 
     render() {
         return (
-            <div>
-                <input type="text" value = {this.state.searchString} onChange = {this.handleSearchBarChange} placeholder="Search your Amish desires..."></input>
-                <button onClick={this.props.submitSearch.bind(this, this.state.searchString)}>search now</button>
+            <div className="searchBarContainer">
+                <div className="row">
+                    <div className="col-md-2"></div>
+                    <div className="col-md-8>">
+                        <div className="input-group">
+                            <span className="input-group-addon">
+                                <select className="categorySelect">
+                                    <option>All</option>
+                                    <option>Clothing</option>
+                                    <option>Cooking</option>
+                                    <option>Books</option>
+                                    <option>Toys</option>
+                                </select>
+                            </span>
+                            <input type="text" className="form-control" value = {this.state.searchString} onChange = {this.handleSearchBarChange} placeholder="Search your Amish desires..."></input>
```

Comment:
We might start running into issues with some of the class names.  It might be good to add "search-" in front of some of them just to make sure they aren't going to conflict with anybody else's classes